.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,68 @@
+# Useful docs:
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Bug Report
+description: File a bug report
+# title: "[Bug]: "
```

Comment:
Were these optional fields in all 3 templates intentionally left behind from Github's examples?

Reply:
Yep. I'll remove them. I already added links to the relevant docs.